README.md

```diff
@@ -37,7 +37,13 @@ The Python data reduction pipeline for WiFeS
    cp /Users/.../pipeline/reduction_scripts/reduce_data.py /Users/.../my_folder/
    cp /Users/.../pipeline/reduction_scripts/*.json /Users/.../my_folder/
    ```
-3. Run `reduce_data.py`, giving the raw data directory as an input parameter. from `/Users/.../my_folder/` The pipeline will run both arms automatically and choose the observing mode by checking the headers.
+3. Set the environment variable `PYWIFES_DIR` to your reference data directory:
+    ```sh
+    export PYWIFES_DIR=/Users/.../pipeline/reference_data
+    ```
+    1. If `PYWIFES_DIR` is not set, the pipeline assumes it is located in your program's *install* directory. 
+    That assumption only works if you install with `pip install -e .`
+4. Run `reduce_data.py`, giving the raw data directory as an input parameter. from `/Users/.../my_folder/` The pipeline will run both arms automatically and choose the observing mode by checking the headers.
```

Comment:
All is good regarding the explanation about how to deal with `reference_data` so the pipeline finds it properly. 
However, as this is something the users will have to do just once after the installation instead of every time they reduce some data, shouldn’t this be in the **installation** section after the `pip install .` step? (i.e., line 31).

Reply:
Environment variables don't stick around each time you login (I don't think), I think it's actually needed every time, unless you add it to your .bashrc.

I think I'll move it to the installation section, but I'll suggest adding it to bashrc, with a couple of dotpoints for `pip install -e .` or running the command before running the script.

Reply:
Good! Thank you! Yes, I think it's better in this way.

Reply:
I've made a second attempt, let me know what you think. I feel like it could be clearer, but I can't figure out the best option.